features/wijzigingen.feature

```diff
@@ -0,0 +1,100 @@
+        En op 12 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999992806"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999992806"
+    
+    Scenario: Gebruiker heropent de volgindicatie
+        Gegeven gebruiker "A" op 17 januari 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999994220" met einddatum "2021-03-01"
+        En gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999994220" en geen einddatum
+        En op 12 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999994220"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" een item met de waarde "999994220"
+
+    Scenario: Gebruiker heropent de volgindicatie en wijzigingen vóór het heropenen worden niet gemeld
+        Gegeven gebruiker "A" op 17 januari 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992351" met einddatum "2021-03-01"
+        En op 21 februari 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999992351"
+        En gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992351" en geen einddatum
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999992351"
+
+    Scenario: Gebruiker vraagt expliciet wijzigingen van vóór het zetten van de volgindicatie
+        Gegeven op 4 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999992971"
+        En gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992971" en geen einddatum
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen?vanaf=2021-03-01
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999992971"
+
+    Scenario: Gebruiker verlengt de volgindicatie en eerdere wijzigingen worden wel gemeld
+        Gegeven gebruiker "A" op 17 januari 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "000009945" met einddatum "2021-03-05"
+        En op 21 februari 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "000009945"
+        En gebruiker "A" op 4 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "000009945" en geen einddatum
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" een item met de waarde "000009945"
+
+    Scenario: Meerdere gebruikers volgen zelfde persoon met verschillende einddatum
```

Comment:
Is deze scenario niet dubbel op? In een voorgaande scenario is aangegeven dat een gebruiker alleen wijzigingen krijgt over de periode waarvoor hij een volgindicatie heeft lopen. De volgindicaties van andere gebruikers op eenzelfde persoon heeft toch geen invloed op het resultaat van een andere gebruiker?

Reply:
dit  voorbeeld illustreert dat de (einddatum van de) volgindicatie per gebruiker verschillend kan zijn en dus een gewijzigde persoon geleverd kan worden aan een gebruiker en niet aan een andere.